### PR TITLE
#1340 Enlarge length for c_location.address, ad_user.email, ...

### PR DESCRIPTION
**Pull request by homebeaver**
Hi all,

This is the PR for issue #1340 for 3.9.2 :

- the location addresses are now 120 chars long, old was 60
- the maximum length of email address is specified in RFC 5321

regards EUGen

### CODE REVIEW

File: migration/391lts-392lts/04540_1340_Enlarge_length_for_address.xml [hunk 1]
```diff
@@ -0,0 +1,461 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<Migrations>
+  <Migration EntityType="D" Name="Enlarge length for address, email" ReleaseNo="3.9.2" SeqNo="4540">
+    <Comments>https://github.com/adempiere/adempiere/issues/1340</Comments>
+    <Step SeqNo="10" StepType="AD">
+      <PO AD_Table_ID="276" Action="U" Record_ID="156" Table="AD_Element">
+        <Data AD_Column_ID="58589" Column="FieldLength" oldValue="60">120</Data>
+        <Data AD_Column_ID="84316" Column="UUID" isOldNull="true">89e0090c-5</Data>
```
[the remark this answers is not in the document]
Comment (homebeaver) [reply migration/391lts-392lts/04540_1340_Enlarge_length_for_address.xml]: Hi @yamelsenih - strange for me: all the UUID are stripped to 10 chars?
I'll remove the erroneous changes - UUID remains null

But the stripping is to be investigeted in a separete issue. My prostrges DB gets the right value:

```SQL
SELECT getUUID()
--returns "c5365fe4-6664-11e9-af04-0b12cb27d422"
```

**Comment by yamelsenih** [comment]
Hi @homebeaver any update?

**Comment by homebeaver** [comment]
@yamelsenih - yes - see my commit 10728bb 

The UUID is unchanged - it remains null. This is good for this PR - no erroneous UUID!

bye

**Comment by yamelsenih** [comment]
Thanks @homebeaver please can you create a new pull request with a commit only?

Best regard

**Comment by homebeaver** [comment]
close this - there is a new cumulative PR for the issue

**Pull request closed without merging**